crypto/cipher/aes_icm.c

```diff
@@ -402,7 +401,8 @@ static srtp_err_status_t srtp_aes_icm_encrypt (void *cv,
     return srtp_err_status_ok;
 }
 
-static const char srtp_aes_icm_description[] = "aes integer counter mode";
+static const char srtp_aes_icm_128_description[] = "AES-128 integer counter mode";
+static const char srtp_aes_icm_256_description[] = "AES-256 integer counter mode";
```

Comment:
Oh, well that's a good reason. I had it in my head that we had only disabled 192-bit keys for BoringSSL, but we now have that via #209.  While I don't particularly see the value in 192-bit key lengths, myself, I sure do like consistency.